### PR TITLE
COMMERCE-6741 Only show products available to the channel in the CPDisplayLayout item selector

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Review by dsitu — APPROVED** [review]

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
@alecsloan  can you please rebase and force push?

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
hey @alecsloan can you please fix SF?
thx

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Review by ivicac — APPROVED** [review]

**Comment by ivicac** [comment]
ci:forward

File: modules/apps/commerce/commerce-product-item-selector-web/src/main/java/com/liferay/commerce/product/item/selector/web/internal/CPDefinitionItemSelectorView.java [hunk 1]
```diff
@@ -83,10 +84,16 @@ public void renderHTML(
 			(HttpServletRequest)servletRequest;
 
 		long cpDefinitionId = ParamUtil.getLong(
-			httpServletRequest, "cpDefinitionId");
+			httpServletRequest, CPField.CP_DEFINITION_ID);
+
+		long channelGroupId = ParamUtil.getLong(
+			httpServletRequest, CPField.CHANNEL_GROUP_ID);
+
```
Comment (ivicac) [inline modules/apps/commerce/commerce-product-item-selector-web/src/main/java/com/liferay/commerce/product/item/selector/web/internal/CPDefinitionItemSelectorView.java:91]: @alecsloan this should be sorted :)

**Review by ivicac — CHANGES_REQUESTED** [review]

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Review by ivicac — APPROVED** [review]

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**